pages/multi/[address]/index.tsx

```diff
@@ -116,34 +122,83 @@ const Multipage = () => {
             }}
           />
         )}
-        {!showSendTxForm && !showDelegateTxForm && (
-          <div className="interfaces">
-            <div className="col-1">
-              <MultisigHoldings holdings={holdings} />
-            </div>
-            <div className="col-2">
-              <StackableContainer lessPadding>
-                <h2>New transaction</h2>
-                <p>
-                  Once a transaction is created, it can be signed by the multisig members, and then
-                  broadcast.
-                </p>
-                <Button
-                  label="Create Transaction"
-                  onClick={() => {
-                    setShowSendTxForm(true);
-                  }}
-                />
-                <Button
-                  label="Create Delegation"
-                  onClick={() => {
-                    setShowDelegateTxForm(true);
-                  }}
-                />
-              </StackableContainer>
-            </div>
-          </div>
+        {showUnDelegateTxForm && (
+          <UnDelegationForm
+            address={multisigAddress}
+            accountOnChain={accountOnChain}
+            closeForm={() => {
+              setShowUnDelegateTxForm(false);
+            }}
+          />
+        )}
+        {showReDelegateTxForm && (
+          <ReDelegationForm
+            address={multisigAddress}
+            accountOnChain={accountOnChain}
+            closeForm={() => {
+              setShowReDelegateTxForm(false);
+            }}
+          />
+        )}
+        {showRewardsTxForm && (
+          <RewardsForm
+            address={multisigAddress}
+            accountOnChain={accountOnChain}
+            closeForm={() => {
+              setShowRewardsTxForm(false);
+            }}
+          />
         )}
+        {!showSendTxForm &&
+          !showDelegateTxForm &&
+          !showUnDelegateTxForm &&
+          !showRewardsTxForm &&
+          !showReDelegateTxForm && (
```

Comment:
Can you make a separate prop for that (something like `anyTxFormVisible`) that is automatically updated from those inputs?